<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>analisis_graph</class>
 <widget class="QDialog" name="analisis_graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="controls" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>430</y>
     <width>801</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#controls
{
background-color: rgb(32,29,72);
border: none;
border-radius: 10px;
}</string>
   </property>
   <widget class="QPushButton" name="start_test">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>10</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">#start_test
{
background-color: transparent;
background-image: url(:/demo/images/Demo/Play.png);
background-repeat: none;
background-position: center;
border: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>571</width>
      <height>131</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">#tableWidget
{
border-radius: 10px;
}</string>
    </property>
    <column>
     <property name="text">
      <string>P0</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>P1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>P2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>P3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>P4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>P5</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>P6</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>P7</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>P8</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>P9</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>P10</string>
     </property>
    </column>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>130</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#progressBar
{
background-color: transparent;
border: 2px solid gray;
border-radius: 10px;
}
#progressBar:chunk
{
background-color: rgb(70,154,170);
width: 2px;
margin: 1px;
}</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>123</x>
      <y>49</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Caudal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Presión</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Voltaje AVG</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Voltaje L1-L2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Voltaje L2-L3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Voltaje L3-L1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Amperaje AVG</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Amperaje L1-L2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Amperaje L2-L3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Amperaje L3-L1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Coseno Phi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Frecuencia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>%Valvula</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>123</x>
      <y>76</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string>Caudal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Presión</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Voltaje AVG</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Voltaje L1-L2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Voltaje L2-L3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Voltaje L3-L1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Amperaje AVG</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Amperaje L1-L2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Amperaje L2-L3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Amperaje L3-L1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Coseno Phi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Frecuencia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>%Valvula</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>123</x>
      <y>103</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="currentIndex">
     <number>6</number>
    </property>
    <item>
     <property name="text">
      <string>Caudal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Presión</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Voltaje AVG</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Voltaje L1-L2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Voltaje L2-L3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Voltaje L3-L1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Amperaje AVG</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Amperaje L1-L2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Amperaje L2-L3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Amperaje L3-L1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Coseno Phi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Frecuencia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>%Valvula</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Curva</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>27</y>
     <width>761</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="graph_button">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#graph_button
{
background-color: transparent;
background-image: url(:/demo/images/Demo/Graph Home.png);
background-repeat: none;
background-position: center;
border: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="datewidget" native="true">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>5</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>296</x>
     <y>400</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="full_graph_button">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>150</y>
     <width>51</width>
     <height>111</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#full_graph_button
{
background-color: transparent;
background-image: url(:/demo/images/Demo/Solapa izq.png);
background-repeat: none;
background-position: center;
border: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="xscale">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>380</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimum">
    <number>50</number>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="singleStep">
    <number>50</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>controls</zorder>
  <zorder>graph_button</zorder>
  <zorder>datewidget</zorder>
  <zorder>label</zorder>
  <zorder>full_graph_button</zorder>
  <zorder>xscale</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
